pxd.c

```diff
@@ -1132,7 +1132,7 @@ int pxd_init(void)
 out_misc:
 	misc_deregister(&pxd_miscdev);
 out_fuse:
-	for (j = 0; j < i; ++j) {
+	for (j = 0; j <= i; ++j) {
```

Comment:
Yes it is. But in the for loop above (line: 1090), if call to pxd_context_init() fails, context at i will be uninitialized.